mmaction/datasets/transforms/loading.py

```diff
@@ -263,6 +263,84 @@ def __repr__(self):
         return repr_str
 
 
+@TRANSFORMS.register_module()
+class SampleFramesV2(SampleFrames):
```

Comment:
I think the class name may need to be more informative and reflect the difference between this class and the base `SampleFrames`.